### PR TITLE
Remove obsolete scripts and clean up entry points in setup.py

### PR DESCRIPTION
**Pull request by lerks**
Some were leftovers from 4da507a6c6a0b45b4dc37f5e976da86df38d5a0e,
others were leftovers from long before.

Remove all testing entry points as testing is not something that is
supposed to be done in production.

---
This change is [Reviewable](https://reviewable.io/reviews/cms-dev/cms/740)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -120,10 +120,6 @@ def find_version():
              "scripts/cmsDropDB"],
     entry_points={
         "console_scripts": [
-            "cmsRunTests=cmstestsuite.RunTests:main",
```
Comment (wil93) [inline setup.py]: This one was used (at least) in the travis script, which is now failing

Comment (lerks) [inline setup.py]: -.-"

I guess I'll have to fix the Travis script...

Comment (stefano-maggiolo) [inline setup.py]: Why? I use it, this is not old or unmaintained in any way, don't break it!

**Comment by lerks** [comment]
Done, a one-liner that fixed two issues. Is there anything else or can I push this?

---

Review status: 0 of 3 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cms-dev/cms/740#-:-KgyJY4GWfxc17V1LHXB:b-339rfe)*

**Comment by stefano-maggiolo** [comment]
:lgtm:

---

Review status: 0 of 3 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cms-dev/cms/740#-:-KgyRry4qcamM0medRmY:bnfp4nl)*

**Comment by lerks** [comment]
Pushed as 0d5f41d26a0c4aea33ded49b137e20d53e7821c6.

**Pull request closed without merging**